pkg/models/system_intake_grb_reviewers.go

```diff
@@ -31,15 +31,15 @@ const (
 
 // SystemIntakeGRBReviewer describes
 type SystemIntakeGRBReviewer struct {
-	BaseStruct
-	EUAUserID      string                  `json:"euaUserId" db:"eua_user_id"`
+	BaseStructUser
+	EUAUserID      uuid.UUID               `json:"euaUserId" db:"eua_user_id"`
```

Comment:
could you update the property name from `EUAUserID` to UserID please? (And same for the table definition?)


```suggestion
	UserID uuid.UUID `json:"userId" db:"user_id"`
```
It's worth calling out that MINT embeds a struct for this ([user_id relation](https://github.com/CMSgov/mint-app/blob/feature/EASI-2771_model_plan_change_history/pkg/models/user_id_relation.go)). There is some extra benefit to that, because you can expose the UserAccount in GQL effectively for free because of the `UserAccount` property, as long as GQL returns the same type. This also allows the front end to get the name of the Reviewer in the same query.

Reply:
That is super interesting - we should comb through the EASi codebase and change that where needed!

Reply:
Added in [003cc5e](https://github.com/CMSgov/easi-app/pull/2588/commits/003cc5e6f4b3c4be29f73b3f9b77b84cda564f5b)